__tests__/server.test.js

```diff
@@ -107,3 +107,49 @@ describe("/api/articles", () => {
 		});
 	});
 });
+
+describe("/api/articles/:article_id/comments", () => {
+	describe("GET", () => {
+		test("GET 200, responds with all comments for a given article", () => {
+			return request(app)
+				.get("/api/articles/3/comments")
+				.expect(200)
+				.then(({ body }) => {
+					expect(body).toEqual([
+						{
+							comment_id: 11,
+							body: "Ambidextrous marsupial",
+							article_id: 3,
+							author: "icellusedkars",
+							votes: 0,
+							created_at: "2020-09-19T23:10:00.000Z",
+						},
+						{
+							comment_id: 10,
+							body: "git push origin master",
+							article_id: 3,
+							author: "icellusedkars",
+							votes: 0,
+							created_at: "2020-06-20T07:24:00.000Z",
+						},
+					]);
```

Comment:
We could test what happens when an article does not have associated comments, like article 2 :)